docs/en/get_started.md

```diff
@@ -74,18 +74,10 @@ pip install -U openmim && mim install "mmcls>=1.0.0rc0"
 
 To verify whether MMClassification is installed correctly, we provide some sample codes to run an inference demo.
 
-**Step 1.** We need to download config and checkpoint files.
-
-```shell
-mim download mmcls --config resnet50_8xb32_in1k --dest .
-```
-
-**Step 2.** Verify the inference demo.
-
 Option (a). If you install mmcls from the source, just run the following command:
 
 ```shell
-python demo/image_demo.py demo/demo.JPEG resnet50_8xb32_in1k.py resnet50_8xb32_in1k_20210831-ea4938fc.pth --device cpu
+python demo/image_demo.py demo/demo.JPEG resnet18_8xb32_in1k --device cpu
```

Comment:
We may need to provide the decription of `resnet18_8xb32_in1k` in this doc.

Reply:
And give hint for how to get the avaliable model/config name like `resnet18_8xb32_in1k`